### PR TITLE
swap readme from rst to markdown

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

**Comment by trel** [comment]
this is 'done'

now need to confirm it renders at pypi as well, so some more changes coming - but not in the README.md.

**Comment by korydraughn** [comment]
I can see the README. Looks good.

**Comment by d-w-moore** [comment]
> I can see the README! Looks good.

Yes, looks good to me.

**Comment by trel** [comment]
rendering well at https://test.pypi.org/project/python-irodsclient/1.2.0.dev3/

i think this is done - ready to squash upon eyeballs.

File: irods/version.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 
-__version__ = '1.1.9'
+__version__ = '1.2.0.dev3'
```
Comment (trel) [inline irods/version.py:3]: will change this before merging.  to just 1.2.0 i think... or can leave this here until we're closer to release date.

Comment (trel) [reply irods/version.py:3]: yes, okay, i'll go back to 1.1.9 for this PR.  just had to bump it to be able to push to pypi.

File: setup.py [hunk 2]
```diff
@@ -21,17 +21,22 @@
...
+                   'Development Status :: 5 - Production/Stable',
                    'Programming Language :: Python :: 2.7',
                    'Programming Language :: Python :: 3.4',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
+                   'Programming Language :: Python :: 3.7',
+                   'Programming Language :: Python :: 3.8',
+                   'Programming Language :: Python :: 3.9',
+                   'Programming Language :: Python :: 3.10',
+                   'Programming Language :: Python :: 3.11',
```
Comment (trel) [inline setup.py:39]: can put these into a different PR as well - in a run-up to release and version bump.

removing from here...

**Review by alanking — COMMENTED** [review]
My eyeballs say that this looks good. Note that this and #506 are on a collision course. What say you, @d-w-moore?

I think it's ready for squash regardless.

**Comment by korydraughn** [comment]
I agree. pypi rendering looks good.

**Comment by trel** [comment]
squashed and numbered.

i'm in a race with #506.

**Review by korydraughn — APPROVED** [review]
Pound it!

**Comment by trel** [comment]
#'d
